.azure-pipelines/build/azure-pipelines.ci.yml

```diff
@@ -36,11 +36,11 @@ steps:
     git config --global --list
     git config --system --list
     git checkout master
-    git config user.name "azurepipelines"
-    git config user.email "azurepipelines@microsoft.com"
+    git config user.name "AzurePipelines"
+    git config user.email "AzurePipelines@microsoft.com"
```

Comment:
The pipeline auto-increments the version numbers of several files (the task manifests, the extension manifest, etc.) and pushes those modified files back to source to avoid any future publish issues caused by versioning. 

This ensures those commits are attributed to AZP (which is a cleaner/better reflection vs. using one of our user accounts)